Makefile

```diff
@@ -4,7 +4,9 @@ all: deps releases all_dists site
 
 DISTS = $(notdir $(wildcard dists/*))
 
-NDISTS = $(DISTS:%=nightly-%)
+NIGHTLY_IGNORED := $(shell cat ignored-during-nightly)
+DISTS_FILTERED = $(filter-out $(NIGHTLY_IGNORED),$(DISTS))
+NDISTS = $(DISTS_FILTERED:%=nightly-%)
 
 .PHONY: all all_dists deps nightly
 all_dists: $(DISTS)
```

Comment:
(creating a thread here)
@aschmahmann following up with the failures during nightly builds,

- It looks like nightly is now failing because we use 1.18, go-ipfs & other have not updated quic-go yet => this PR uses 1.17,
- It looks like legacy code builds with 1.18, but figured it make sense to skip legacy code anyway,

Reply:
go-ipfs is now building with 1.18 and I'd rather not hold back just because some others haven't yet (e.g. we can get out some PRs quickly to update the relay daemon and ipget, and @hsanjuan will cover the cluster next time he does a release).

In the meanwhile having `ignore-during-nightly` means we can do things like ignore cluster builds and file an issue indicating we should update cluster and remove the ignore.